docs/how-to/add-model-and-api-endpoints.md

```diff
@@ -1,12 +1,56 @@
 # Add new model and API endpoints
 
-This guide aims to enable python developers with little or no django experience to be able to add django models and API endpoints to the project.
-
-First, we identify an issue to work on from the [Onboarding page](https://github.com/hackforla/peopledepot/wiki/Developer-Onboarding). Let's say we want to work on the [recurring_event issue](https://github.com/hackforla/peopledepot/issues/14). Then, we follow the this guide.
-
-## Add the model in django
+This guide aims to enable developers with little or no django experience to add django models and API endpoints to the project. Most code examples are followed by detailed explanations.
+
+The developer will have exposure to the following in this document:
+
+- python
+- django
+- django rest framework
+- relational database through the Django ORM (object-relational mapper)
+- data types
+- object-oriented concepts (object, inheritance, composition)
+- unit testing
+- API design
+- command line
+
+This guide assumes the developer has followed the [contributing doc](https://github.com/hackforla/peopledepot/blob/main/docs/CONTRIBUTING.md#310-push-to-upstream-origin-aka-your-fork) up to section 3.2 and have created a local branch to work on this. The development server would be already running in the background and will automatically apply the changes when we save the files.
+
+We will choose the [recurring_event issue](https://github.com/hackforla/peopledepot/issues/14) as an example. Our goal is to create a database table and an API that a client can use to work with the data. The work is split into 3 testable components: the model, the admin site, and the API
+
+Let's start!
+
+## The model
+
+### Add the model in django
+
```

Comment:
Open `peopledepot/app/core/models.py`

// It could be helpful for some?

Reply:
Added the file path